### ISSUE TITLE
LoginDialog Hints on Android

### ISSUE THREAD

**Issue by nathan-c-jones** [NativeScript/NativeScript#6401]
**Is your feature request related to a problem? Please describe.**
LoginDialog on Android does not have hints for the text fields.

**Describe the solution you'd like**
Hints added to the text fields on Android (iOS uses Login and Password for the hints).

**Comment by NickIliev** [comment]
TODO:
- [ ] Add documentation examples once the PR is merged into `master`

**Comment by NickIliev** [comment]
TODO:
- [ ] Add documentation examples once the PR is merged into `master`

**Comment by NickIliev** [comment]
Docs added via [https://github.com/NativeScript/nativescript-sdk-examples-ng/pull/332](https://github.com/NativeScript/nativescript-sdk-examples-ng/pull/332) and via [https://github.com/NativeScript/nativescript-sdk-examples-js/pull/75](https://github.com/NativeScript/nativescript-sdk-examples-js/pull/75)
